Sample Applications/ExpenseIt/ExpenseItDemo/CreateExpenseReportDialogBox.cs

```diff
@@ -43,5 +44,18 @@ private void cancelButton_Click(object sender, RoutedEventArgs e)
         {
             DialogResult = false;
         }
+
+        private void expenseDataGrid1_AutoGeneratingColumn(object sender, DataGridAutoGeneratingColumnEventArgs e)
+        {
+            if (e.Column.Header.ToString() == "Type")
+            {
+                e.Column.Header = "Expense Type";
+            }
+            if (e.Column.Header.ToString() == "Cost")
+            {
+                e.Column.HeaderStyle = FindResource("ColumnHeaderStyle2") as Style;
```

Comment:
Rename ColumnHeaderStyle2 to a more descriptive name

Reply:
fixed names in Styles.xaml